tigerpath/majors_and_certificates/majors/GEO_2018.json

```
@@ -0,0 +1,467 @@
+            "GEO 203"
+          ]
+        }
+      ]
+    },
+    {
+      "name": "Tracks",
+      "max_counted": 1,
+      "min_needed": 1,
+      "explanation": "There are five department specializations: Microbiology (MB), Environmental Geochemistry (EC), Oceans, Atmosphere, & Climate (OA), Geology & Earth History (GE), and Geophysics (GP). Each specialization has a different set of prerequisites (1,2), basic science requirements (3), geoscience core requirements (4) & recommended electives (5) as follows",
+      "req_list": [
+        {
+          "name": "Microbiology",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": null,
+          "req_list": [
+            {
+              "name": "General Requirements",
+              "max_counted": 1,
+              "min_needed": "ALL",
+              "explanation": null,
+              "req_list": [
+                {
+                  "name": "Statistics",
+                  "max_counted": 1,
+                  "min_needed": 1,
+                  "explanation": "One statistics requirement, to be completed by the end of the junior year (GEO 422 or ORG 245).",
+                  "completed_by_semester": 6,
+                  "course_list": [
+                    "GEO 422",
+                    "ORF 245"
+                  ]
+                },
+                {
+                  "name": "Math and Science",
+                  "max_counted": 1,
+                  "min_needed": "ALL",
+                  "explanation": "Four required core math and science courses that vary depending on the chosen specialization. ISC 231-234 and/or AP credit may substitute for some of these courses. Students interested in graduate school are encouraged to take more than these minimum basic science requirements.",
+                  "course_list": [
+                    "EEB 211",
+                    "MOL 214",
+                    "CHM 201",
+                    "CHM 202"
+                  ]
+                },
+              ]
+            },
+            {
+              "name": "Departmental Requirements",
+              "max_counted": 1,
+              "min_needed": "ALL",
+              "explanation": null,
+              "req_list": [
+                {
+                  "name": "Core Geoscience",
+                  "max_counted": 1,
+                  "min_needed": "ALL",
+                  "explanation": "Two core geoscience requirements that vary depending on the chosen specialization.",
+                  "course_list": [
+                    "GEO 363",
+                    "GEO 417"
+                  ]
+                },
+                {
+                  "name": "Electives",
+                  "max_counted": 1,
+                  "min_needed": 5,
+                  "explanation": "Five elective 300/400 level geosciences courses.",
+                  "course_list": [
+                    "GEO 3*",
+                    "GEO 4*"
+                  ]

```

We should probably encode the recommended electives for each specialization, since the department provides them on the webpage.
```suggestion
                  "course_list": [
                    "GEO 362",
                    "GEO 416",
                    "GEO 418",
                    "GEO 428",
                    "MOL 345",
                    "MOL 380"
                  ]
```

---

I didn't encode the recommended electives because it looked like part of a suggested or example set of courses a student could take in contrast to some of the required courses that change by track like "Required Core Math and Science" and "Required Core Geoscience", which explicitly state they're required.

---

Maybe we can encode GEO 3* and GEO 4* and also include the extra electives they mention from other departments?

---

Yeah, I guess it ends up being a matter of how strongly these courses are "recommended". That is, is it really just a sample list, or, on the other hand, do students need to get departmental approval in order to count any courses not on the list?

Your suggestion works great for now, so I'll approve it, but I guess we could probably contact the department to find out what their policy is.